lib/index.d.ts

```diff
@@ -0,0 +1,68 @@
+import { JSONSchema7 } from 'json-schema';
+export interface Abi {
```

Comment:
Is this file code generated or handwritten?

```suggestion
export interface AbiRoot {
```

Reply:
Handwritten